tensorflow_cloud/validate.py

```diff
@@ -150,6 +153,53 @@ def _validate_cluster_config(chief_config, worker_count, worker_config):
             "Received {}.".format(worker_count)
         )
 
+def _validate_labels(labels):
+    """Validates labels conform guidelines at 
+       https://cloud.google.com/ai-platform/training/docs/resource-labels"""
+    if not labels:
+        print("No label for the job. Consider creating labels"
+              "for easier organizing submitted jobs.")
+
+    if len(labels) > 64:
```

Comment:
AI platform specific label validation should live in gcp.py. We can just invoke that function from here.

Reply:
Thanks for pointing out. Moved.